src/Maths/Silk.NET.Maths/Scalar.Inverse.cs

```diff
@@ -0,0 +1,153 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Runtime.CompilerServices;
+using static Silk.NET.Maths.Helper;
+
+namespace Silk.NET.Maths
+{
+    public static partial class Scalar
+    {
+        /// <summary>
+        /// Calculates the inverse of a number.
+        /// </summary>
+        /// <param name="x">The number.</param>
+        /// <typeparam name="T">The type of <paramref name="x"/>.</typeparam>
+        /// <returns>The inverse of the given number.</returns>
+        [MethodImpl(MaxOpt)]
+        public static T Inverse<T>(T x) where T : unmanaged
```

Comment:
Should this be reciprocal?

Discussion point.

Reply:
reciprocal may be a better name. I've just put this in there to allow for easy optimization down the road.